tools/commands/config.py

```diff
@@ -113,15 +123,16 @@ def __guess_setup_id(self):
         try:
             from chdlab.commands.config import CHDLAB_config
             return CHDLAB_config().setup_number
-        except:
+        except Exception:
```

Comment:
The previous commit still leaves a catch-all except, so why not do the same here?

Reply:
flake8 compains with `E722 do not use bare except`, so I add `Exception`